app/sessionStore.js

```diff
@@ -675,6 +675,28 @@ module.exports.runPreMigrations = (data) => {
 }
 
 module.exports.runPostMigrations = (data) => {
+  // fingerprinting protection migration
```

Comment:
Not that we have to do it now... but I'm Curious what you think @diracdeltas and @darkdh 

Similar to Ruby and Python, maybe we could create a `/migrations/` folder and have a single file (ex: `0-20-0_1st_3rd_fingerprint`) with a two possible functions (`postMigrate`, `preMigrate`) for each type of migration that we'd like to run. 

We could then call them like so:
```
migrations = [
  require('../../migrations/0-12-0_search_xml'),
  require('../../migrations/0-14-0_autofill_data'),
  require('../../migrations/0-20-0_1st_3rd_fingerprint')
]
migrations.runPostMigrations()
```